packages/core/src/components/FormLabel/FormLabel.example.jsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/display-name */
+import FormLabel from './FormLabel';
+import React from 'react';
+
+export default function() {
+  return (
+    <FormLabel
```

Comment:
Can we add `requirementLabel` to this component to provide an example of what this looks like?